ansible_builder/main.py

```diff
@@ -428,12 +430,18 @@ def prepare_build_stage_steps(self):
         ])
         return self.steps
 
-    def prepare_final_stage_steps(self):
+    def prepare_final_stage_steps(self, collections):
         self.steps.extend([
             "",
             "FROM $ANSIBLE_RUNNER_IMAGE"
             "",
         ])
+        if collections:
+            self.steps.extend([
+                "",
+                "LABEL collections='{}'".format(json.dumps(collections)),
+                "",
+            ])
```

Comment:
Another idea, is to remove this from the Dockerfile, then use the --label flag on the command line. That will atleast remove this blob from the file, and each time ansible-builder runs, it gets setup.

Then users or docker / podman would still need to pass the format some how.

Reply:
I think you must have left this comment before we spoke in Slack, but for posterity I'll reiterate here: we don't have control over the invocation of `docker build` in our downstream build system.